apps/calendar/playwright/day/timeGridEventMoving.e2e.ts

```diff
@@ -162,3 +162,37 @@ test.describe('CSS class for a move event', () => {
     expect(await moveEventClassLocator.count()).toBe(0);
   });
 });
+
+const [LONG_TIME_EVENT] = mockDayViewEvents.filter(({ title }) => title === 'long time');
```

Comment:
```suggestion
const LONG_TIME_EVENT = mockDayViewEvents.find(({ title }) => title === 'long time');
```

어차피 첫번째 이벤트 하나만 필요하니까 find로 찾아도 되겠네요.

Reply:
`find`로 찾게되면 `undefined`가 나올 수 있어서(없는걸 알지만 타입상) 추가 작업이 필요하게되서 유지하도록 하겠습니다!